src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -150,6 +150,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         { ShortcutAction::MoveTab, MoveTabArgs::FromJson },
         { ShortcutAction::ToggleCommandPalette, ToggleCommandPaletteArgs::FromJson },
         { ShortcutAction::FindMatch, FindMatchArgs::FromJson },
+        { ShortcutAction::NewWindow, NewWindowArgs::FromJson },
```

Comment:
This line seems to be the only relevant change in this file. All of the other ones are just moving stuff around.